lib/BackgroundJob/ExpireUsers.php

```diff
@@ -195,14 +229,14 @@ protected function setCreatedAt(IUser $user, int $time): void {
 
 	protected function prepareLDAPUser(IUser $user): bool {
 		try {
-			$ldapProvider = $this->server->query('LDAPProvider');
+			$ldapProvider = $this->server->get('LDAPProvider');
```

Comment:
you should be able to request `ILDAPProvider::class`

Reply:
Thanks, adjusted